res/values/strings.xml

```diff
@@ -465,11 +465,10 @@
     <!-- share and forward messages -->
     <!-- Translators: Title shown above a chat/contact list; the user selects the recipient of the messages he wants to forward to -->
     <string name="forward_to">Forward to…</string>
-    <string name="share_multiple_attachments">Send %1$d files to the selected chat?\n\nThe files are sent unmodified in their original size, eg. images and videos are not recoded.</string>
-    <string name="share_multiple_attachments_multiple_chats">Send %1$d file(s) to %2$d chats?\n\nThe files are sent unmodified in their original size, eg. images and videos are not recoded.</string>
+    <string name="share_multiple_attachments">Send %1$d file(s) unmodified to the selected chat?</string>
+    <string name="share_multiple_attachments_multiple_chats">Send %1$d unmodified file(s) to %2$d chats?</string>
```

Comment:
not sure if it is useful to leave out details. wondering if there was some feedback before.

Reply:
Is there an option to send them compressed or without metadata?
If not, sending files is sending files and "unmodified" can be omitted.